spec/features/team/invite_a_team_member_spec.rb

```diff
@@ -64,14 +66,40 @@
       end
 
       context "with an email that already exists" do
-        let(:invited_user_email) { user.email }
+        context "and is confirmed" do
```

Comment:
contexts should always start with "when", "with" or "without": http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording